bpf/lib/nodeport_egress.h

```diff
@@ -105,6 +105,8 @@ int tail_handle_snat_fwd_ipv6(struct __ctx_buff *ctx)
 
 #ifdef IS_BPF_OVERLAY
 	obs_point = TRACE_TO_OVERLAY;
+#elif defined(IS_BPF_WIREGUARD)
+	obs_point = TRACE_TO_CRYPTO;
 #else
 	obs_point = TRACE_TO_NETWORK;
 #endif
```

Comment:
I wonder if this should just be a macro that is defined by all programs which include `nodeport_egress.h`?

```
#define NODEPORT_OBS_POINT_EGRESS      TRACE_TO_CRYPTO
```

Reply:
> I wonder if this should just be a macro that is defined by all programs which include nodeport_egress.h?

Mmm what about moving the define at the top of `nodeport_egress.h`?
```c
#if defined(IS_BPF_OVERLAY)
#define NODEPORT_OBS_POINT_EGRESS      TRACE_TO_OVERLAY
#elif defined(IS_BPF_WIREGUARD)
#define NODEPORT_OBS_POINT_EGRESS      TRACE_TO_CRYPTO
#else
#define NODEPORT_OBS_POINT_EGRESS      TRACE_TO_NETWORK
#endif
```

Reply:
works for me!